scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -53,6 +53,12 @@ def call(final pipelineContext) {
     [
       stageName: 'Java 8 Smoke', target: 'test-junit-smoke-jenkins', javaVersion: 8, timeoutValue: 20,
       component: pipelineContext.getBuildConfig().COMPONENT_JAVA
+    ],
+    [
```

Comment:
This will of course disapper before merging.

Reply:
oh, okay!